recipes/example3/meta.yaml

```diff
@@ -10,7 +10,7 @@
 #  `conda install openssl -c conda-forge``
 
 package:
-  name: {{ name|lower }}
+  name: simplejson2
```

Comment:
Is this on purpose?

Reply:
Nope, I'll be reverting the examples once I know this works.